arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts

```diff
@@ -9,7 +9,7 @@
 
 / {
 	model = "Pine64 Ox64";
-	compatible = "sipeed,m1s", "bflb,bl808";
+	compatible = "pine64,ox64", "bflb,bl808";
```

Comment:
Some of these changes are unrelated to the pinctrl driver, it will likely make integration with buildroot_bouffalo if you refactor them into a separate patch